src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -31,7 +31,7 @@ private int GetNextAvailableId(StoreView snapshot)
 
         internal override void Initialize(ApplicationEngine engine)
         {
-            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MinimumDeploymentFee), new StorageItem(100_00000000));
+            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MinimumDeploymentFee), new StorageItem(1_00000000));
```

Comment:
10GAS?

Reply:
Done. It's good for me, better than 100GAS.